### PR TITLE
Project skeleton

### PR DESCRIPTION
**Pull request by serhii-lisniakov**
includes components skeletons for header, body and footer

### CODE REVIEW

File: src/app/app-routing.module.ts [hunk 1]
```diff
@@ -1,7 +1,15 @@
 import { NgModule } from '@angular/core';
```
Comment (iyarymovych-reenbit) [inline src/app/app-routing.module.ts:1]: Make compatible all imports.
I mean with spaces in brackets or without - decide.

File: package.json [hunk 2]
```diff
@@ -15,10 +15,12 @@
     "@angular/common": "~10.2.0",
```
Comment (iyarymovych-reenbit) [inline package.json]: Could you please make sure that all TS Lint issues are fixed in your project?

File: src/app/app.component.html [hunk 3]
```diff
@@ -1,6 +1,7 @@
 <div class="container">
```
Comment (iyarymovych-reenbit) [inline src/app/app.component.html:1]: Where is your router outlet?

File: src/app/body/body.component.html [hunk 4]
```diff
@@ -0,0 +1,3 @@
+<main class="main">
```
Comment (iyarymovych-reenbit) [inline src/app/body/body.component.html]: You don't need this component, router outlet move to the app component.

File: src/app/footer/footer-tags/footer-tags.component.html [hunk 5]
```diff
@@ -0,0 +1,15 @@
+<div class="footer-tags_container">
```
Comment (iyarymovych-reenbit) [inline src/app/footer/footer-tags/footer-tags.component.html:1]: Beautify your templates - you don't need so mane unneeded empty lines.

File: src/app/app-routing.module.ts (hunk 1 above)
Comment (serhii-lisniakov) [reply src/app/app-routing.module.ts:1]: fixed

File: package.json (hunk 2 above)
Comment (serhii-lisniakov) [reply package.json]: fixed

File: src/app/body/body.component.html (hunk 4 above)
Comment (serhii-lisniakov) [reply src/app/body/body.component.html]: fixed

File: src/app/footer/footer-tags/footer-tags.component.html (hunk 5 above)
Comment (serhii-lisniakov) [reply src/app/footer/footer-tags/footer-tags.component.html:1]: fixed

**Review by iyarymovych-reenbit — APPROVED** [review]

**Pull request merged**